package.json

```diff
@@ -38,12 +39,12 @@
     "grunt": "^1.0.4",
     "grunt-env": "^1.0.1",
     "grunt-eslint": "^22.0.0",
-    "grunt-jsdoc-to-markdown": "^5.0.0",
+    "grunt-jsdoc-to-markdown": "^6.0.0",
     "grunt-mocha-istanbul": "^5.0.2",
     "grunt-mocha-test": "^0.13.3",
     "istanbul": "^0.4.5",
     "jit-grunt": "^0.10.0",
-    "mocha": "^7.0.1",
+    "mocha": "^10.2.0",
```

Comment:
Would you mind undoing these? I know the devDepencies are out of date, but these sorts of changes should be done separately.